packages/frontend/src/stores/conversations.ts

```diff
@@ -0,0 +1,36 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Readable } from 'svelte/store';
+import { readable } from 'svelte/store';
+import { Messages } from '@shared/Messages';
+import { rpcBrowser, studioClient } from '/@/utils/client';
+import type { Conversation } from '@shared/src/models/IPlaygroundMessage';
+
+export const conversations: Readable<Conversation[]> = readable<Conversation[]>([], set => {
```

Comment:
Is there a reason why you did not use `RPCReadable`?

```
export const conversations = RPCReadable<Conversation[]>(
[],
[Messages.MSG_CONVERSATIONS_UPDATE],
studioClient.getPlaygroundConversations,
)
```

Reply:
because of the debouncer, using the RPCReadable here breaks the user experience. It will receive the tokens only every 10s or so.

Reply:
Oh I see, i've never look up the implementation, after reading it, it makes sense.

Reply:
Please add a comment to explain it in the code